### ISSUE TITLE
Feature Request: Execute TTP Variant with API

### ISSUE THREAD

**Issue by ajpc500** [preludeorg/operator-support#436]
A technique's variants can currently be executed via the UI, but not via the API.

**Comment by privateducky** [comment]
Good ask. This has been on my mind lately - I want to create a more fluid way to select variants for TTPs from within the UI, which would naturally work for the API as well. 

In the API today, you can pass in either a TTP identifier (UUID) or the contents of a full YML file itself (swagger snippet below). This would require reordering the YML file "by hand" before submitting it to the endpoint, which isn't ideal. 

Expect a solution that does what you're looking for + solves this ticket: https://github.com/preludeorg/operator-support/issues/356

```
                /*
                    #swagger.description = 'Queue a new instruction up for an agent to collect and execute'
                    #swagger.parameters['name'] = {
                        in: "path",
                        description: 'The display name of an existing agent',
                        required: true,
                    }
                    #swagger.parameters['instruction'] = {
                        in: "body",
                        description: 'An instruction object to queue',
                        required: true,
                        type: "object",
                        schema: {
                            $instruction: {
                                $id: "",
                                adversary: "",
                                $ttp: "",
                                $facts: [],
                                $executor: ""
                            }
                        }
                    }
                */
```

**Comment by privateducky** [comment]
This is now in flight and will be available in 1.5.

**Comment by ajpc500** [comment]
Fantastic, cheers!
